doc/_data/scripts/gen_federation_md.py

```diff
@@ -0,0 +1,56 @@
+"""Generate snippets of markdown for various types of supporters.
+These are meant to be inserted into our docs.
+"""
+import pandas as pd
+from pathlib import Path
+from yaml import safe_load
+
+# Code to generate the HTML grid
+template_binderhub = """
+```{{grid-item-card}}
+:text-align: center
+:class-header: bg-light
+:class-body: sd-p-4 sd-m-auto
+:class-footer: bg-light
+:text-align: center
+:link: {URL_BINDERHUB}
+
+**{BINDERHUB_SUBDOMAIN}**
+
+^^^
+
+<img src="{LOGO}" style="max-height: 5em; min-height: 2em;" />
+```
+
+"""
+
+# Read from our YAML data file 
+path_root = Path(__file__).parent.parent
+path_data = path_root / "support" / "federation.yml"
+binderhubs = pd.DataFrame(safe_load(path_data.read_text()))
+
+# Generate markdown entries for each federation member
+entries = []
+for ix, binderhub in binderhubs.iterrows():
```

Comment:
As with supporters I think this can be simplified to avoid pandas
```py
binderhubs_yml = safe_load(path_data.read_text())
for binderhub in binderhubs_yml:
   ...
```